package.json

```diff
@@ -27,6 +27,7 @@
     "@octokit/rest": "^18.0.0",
     "@slack/webhook": "^5.0.4",
     "@types/node-fetch": "^2.5.7",
+    "tsc": "^1.20150623.0",
```

Comment:
I think we only want this in the dev-dependencies. You can install it in there with `npm install --save-dev tsc`

Reply:
ah sorry! i'll remove these :)